src/player-data.ts

```diff
@@ -119,7 +120,7 @@ export default class PlayerData extends EventEmitter {
       case Resource.ChargePower: count > 0 ? this.chargePower(count) : this.spendPower(-count); return;
       case Resource.Range: this.range += count; return;
       case Resource.TemporaryRange: this.temporaryRange += count; return;
-      case Resource.GaiaFormer: this.gaiaformers += count; return;
+      case Resource.GaiaFormer: count > 0 ? this.gaiaformers += count : this.gaiaformersInGaia -= count; return;
```

Comment:
`this.gaiaformers += count` should be done always. Then, if count < 0, ` this.gaiaformersInGaia -= count`.